apps/content/management/commands/estimate_ayah_slicing_size.py

```
@@ -0,0 +1,199 @@
+
+    warn_object_count = getattr(settings, "AYAH_SLICING_WARN_OBJECT_COUNT", None) or None
+    warn_estimated_bytes = getattr(settings, "AYAH_SLICING_WARN_ESTIMATED_BYTES", None) or None
+
+    return {
+        "asset_count": asset_count,
+        "folder_count": folder_count,
+        "track_count": track_count,
+        "timing_count": timing_count,
+        "expected_object_count": timing_count,
+        "total_source_bytes": total_source_bytes,
+        "estimated_output_bytes": int(round(estimated_output_bytes)),
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)